deck-builder/main.js

```diff
@@ -321,17 +321,734 @@ var boosterPacksList = [
+        flavorText: 'Kayaba: "However, if you have any feelings left for ' +
+          'that world besides hatred……"',
+        previewPic: 'booster-packs/sao-vol-2-card-list/card-preview/s26-e078-preview.jpg',
+        picture: 'booster-packs/sao-vol-2-card-list/card/s26-e078.jpg'
+      },
+      {
+        id: 's26-e079',
+        class: 'sword-art-online-vol-2-bp',
+        cardName: 'Confrontation Between the Strongest Players',
+        cardNumber: 'SAO/S26-E079',
+        expansion: 'Sword Art Online Vol. 2',
+        cardType: 'Climax',
+        level: '',
+        power: '',
+        trigger: 'booster-packs/card-triggers/gate.png',
+        rarity: 'CR',
+        side: 'S',
+        color: 'blue',
+        cost: '',
+        soul: '',
+        attribute: [],
+        text: '【CONT】 All of your characters get +1000 power and +1 soul.\n' +
+          '（【GATE】: When this card triggers, you may choose a climax in ' +
+          'your waiting room, and return it to your hand.)',
+        flavorText: 'Haaaa!',
+        previewPic: 'booster-packs/sao-vol-2-card-list/card-preview/s26-e079-preview.jpg',
+        picture: 'booster-packs/sao-vol-2-card-list/card/s26-e079.jpg'
+      },
+      {
+        id: 's26-e080',
+        class: 'sword-art-online-vol-2-bp',
+        cardName: 'Reunion with Yui',
+        cardNumber: 'SAO/S26-E080',
+        expansion: 'Sword Art Online Vol. 2',
+        cardType: 'Climax',
+        level: '',
+        power: '',
+        trigger: 'booster-packs/card-triggers/double-soul.png',
+        rarity: 'CC',
+        side: 'S',
+        color: 'blue',
+        cost: '',
+        soul: '',
+        attribute: [],
+        text: '【AUTO】 When this card is placed on your climax area from ' +
+          'hand, draw a card, choose one of your characters, and that ' +
+          'character gets +2000 power and +1 soul until end of turn.',
+        flavorText: 'Kirito: "Ah……!?"',
+        previewPic: 'booster-packs/sao-vol-2-card-list/card-preview/s26-e080-preview.jpg',
+        picture: 'booster-packs/sao-vol-2-card-list/card/s26-e080.jpg'
+      }
+    ]
   }
 ]
 
 var $boosterPacksSection = document.querySelector('#booster-packs-section')
+var $displayPacksSelected = document.querySelector('#display-packs-selected')
 var $cardListSection = document.querySelector('#card-list-section')
 var $cardList = document.querySelector('#card-list')
-var $displayPacksSelected = document.querySelector('#display-packs-selected')
+var $cardCountSection = document.querySelector('#card-count-section')
+var $characterCounter = document.querySelector('#character-counter')
+var $eventCounter = document.querySelector('#event-counter')
+var $climaxCounter = document.querySelector('#climax-counter')
+var $cardCounter = document.querySelector('#card-counter')
 var $confirm = document.querySelector('#confirm')
 var $returnBpSection = document.querySelector('#return-bp-section')
 var $selectedPacks = []
+var $deckList = []
```

Comment:
Data should not be prefixed with a `$`. We use that to differentiate DOM variables from data variables.